Sources/MapboxDirections/RouteRefreshing.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+
+/**
+ A skeletal route containing infromation to refresh `Route` object annotations.
+ */
+public protocol RouteRefreshing {
+    var refreshedLegs: [RouteLegRefreshing] { get }
+}
+
+/**
+ A skeletal route leg containing infromation to refresh `RouteLeg` object annotations.
```

Comment:
"annotations" or "attributes"? (here an in the comments above)

Reply:
These are synonyms.

Reply:
A bit confusing to me as a non-native speaker

Reply:
Corrected